Готовое приложение с мок данными по условиям 4-го спринта

Dimaulta opened the pull request:
+ шрифт

Dimaulta committed:
Готовое приложение с мок данными по условиям 4-го спринта
+ шрифт

DanilaG commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,170 @@
+        if isCorrect {
+            correctAnswers += 1
+        }
+        
+        
```
Обычно стараются делать не более 1 вертикального отступа. Давай удалим лишний

DanilaG commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,170 @@
+    }
+    
+    private func showAnswerResult(isCorrect: Bool) {
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 8
```
Данные параметры не зависят от результата ответа. Их можно выставить единожды во viewDidLoad

Dimaulta committed:
1. Исправлено критическое замечание (добавлено скругление углов у постеров)
2. Добавлено isEnabled на случай если пользователь слишком часто будет нажимать на кнопки

Dimaulta commented:
@DanilaG я исправил критическое замечание в учебном проекте (скруглкние углов постеров)
